libs/Microsoft.MixedReality.WebRTC.Native/include/interop_api.h

```diff
@@ -431,24 +594,77 @@ enum class VideoProfileKind : int32_t {
   kVideoHdr8,
 };
 
-/// Register a callback fired when an audio frame is available from a local
-/// audio track, usually from a local audio capture device (local microphone).
-///
-/// -- WARNING --
-/// Currently this callback is never fired, because the internal audio capture
-/// device implementation ignores any registration and only delivers its audio
-/// data to the internal WebRTC engine for sending to the remote peer.
-MRS_API void MRS_CALL mrsPeerConnectionRegisterLocalAudioFrameCallback(
-    PeerConnectionHandle peerHandle,
-    PeerConnectionAudioFrameCallback callback,
-    void* user_data) noexcept;
+enum class mrsTransceiverStateUpdatedReason : int32_t {
+  kLocalDesc,
+  kRemoteDesc,
+  kSetDirection
+};
 
-/// Register a callback fired when an audio frame from an audio track was
-/// received from the remote peer.
-MRS_API void MRS_CALL mrsPeerConnectionRegisterRemoteAudioFrameCallback(
-    PeerConnectionHandle peerHandle,
-    PeerConnectionAudioFrameCallback callback,
-    void* user_data) noexcept;
+/// Flow direction of the media inside the transceiver. This maps to whether
+/// local and/or remote tracks are attached to the transceiver. The local
+/// track corresponds to the send direction, and the remote track to the
+/// receive direction.
+enum class mrsTransceiverDirection : int32_t {
+  kSendRecv = 0,
+  kSendOnly = 1,
+  kRecvOnly = 2,
+  kInactive = 3
+};
+
+/// Same as |mrsTransceiverDirection|, but including optional unset.
+enum class mrsTransceiverOptDirection : int32_t {
+  kNotSet = -1,
+  kSendRecv = 0,
+  kSendOnly = 1,
+  kRecvOnly = 2,
+  kInactive = 3
+};
+
+/// Configuration for creating a new audio transceiver.
+struct AudioTransceiverInitConfig {
+  /// Mandatory name of the audio transceiver. This must be a valid SDP token;
+  /// see |mrsSdpIsValidToken()|.
+  const char* name = nullptr;
+
+  /// Initial desired direction of the transceiver media when created.
+  mrsTransceiverDirection desired_direction =
+      mrsTransceiverDirection::kSendRecv;
+
+  /// Semi-colon separated list of stream IDs associated with the transceiver.
+  /// Use |Transceiver::DecodeStreamIDs()| and |Transceiver::EncodeStreamIDs()|.
```

Comment:
nit: shouldn't mention private functions in the public API.

Reply:
Fixed.

Reply:
Also found mention of Encode/DecodeIceServers.

Reply:
Fixed too.